db/schema.rb

```diff
@@ -11,12 +11,28 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180416231107) do
+ActiveRecord::Schema.define(version: 20180423182604) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
 
+  create_table "async_transactions", force: :cascade do |t|
+    t.string   "type"
+    t.string   "user_uuid"
+    t.string   "source_id"
+    t.string   "source"
+    t.string   "status"
+    t.string   "transaction_id"
+    t.string   "transaction_status"
+    t.datetime "created_at",         null: false
+    t.datetime "updated_at",         null: false
+  end
+
+  add_index "async_transactions", ["source_id"], name: "index_async_transactions_on_source_id", using: :btree
+  add_index "async_transactions", ["transaction_id"], name: "index_async_transactions_on_transaction_id", using: :btree
+  add_index "async_transactions", ["user_uuid"], name: "index_async_transactions_on_user_uuid", using: :btree
+
```

Comment:
We'll also probably want indexes on our common multi-column queries for the transaction status GET endpoints:

`source, source_id, transaction_id`
and
`source, user_uuid`

since we'll almost always be querying those together. We can discuss in a call though.